internal/test/flaky.go

```diff
@@ -0,0 +1,16 @@
+package test
+
+import (
+	"os"
+	"testing"
+)
+
+// Flaky will skip the test if the FLAKY environment variable is not empty.
+func Flaky(t *testing.T) {
+	// We can't use flags because it fails for tests that does not import this package
+	if os.Getenv("FLAKY") != "" {
```

Comment:
shouldn't this be `==` ?

Reply:
I did the same mistake originally, not it must not.
This is the `return` path, so not `Skip`.
We want to run the test when `FLAKY=1`. `"1" != "" => true` so we `return`, *without* skipping.

Reply:
ah yeah, maybe a better name then since we both made the same mistake

Reply:
Renamed to `RUN_FLAKY_TEST`